WcaOnRails/spec/factories/committee_positions.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :committee_position do
+    sequence(:name) { |n| "Position #{n}" }
+    slug { name.parameterize }
```

Comment:
Good point. Only after create not build. And I guess that's how it happens so I'll take this out of the factory.